languages/erlang/reference/implementing-a-concept-exercise.md

```diff
@@ -1,5 +1,72 @@
 # How to implement an Erlang concept exercise
 
-TODO: describe how to implement a concept exercise for the Erlang track. For inspiration, check out the [C# version of this file][csharp-implementing].
+This document describes how to implement a concept exercise for the Erlang track. As this document is generic, the following placeholders are used:
 
-[csharp-implementing]: ../../csharp/reference/implementing-a-concept-exercise.md
+- `<SLUG>`: the name of the exercise in kebab-case (e.g. `anonymous-functions`).
+- `<NAME>`: the name of the exercise in snake_case (e.g. `anonymous_functions`).
+
+Before implementing the exercise, please make sure you have a good understanding of what the exercise should be teaching (and what not). This information can be found in the exercise's GitHub issue. Having done this, please read the [Erlang concept exercises introduction][concept-exercises].
+
+To implement a concept exercise, the following files must be created:
+
+```
+languages
+└── erlang
+    └── exercises
+        └── concept
+            └── <SLUG>
+                ├── .docs
+                │   ├── instructions.md
+                │   ├── introduction.md
+                │   ├── hints.md
+                │   └── after.md (optional)
```

Comment:
The `after.md` is not optional, and we've since added a `source.md` file (which _is_ optional in certain cases):

```suggestion
                │   ├── instructions.md
                │   ├── introduction.md
                │   ├── hints.md
                │   ├── after.md
                │   └── source.md (required if there are third-party sources)                
```

           |   ├── after.md
                |   ├── instructions.md
                |   ├── introduction.md
                |   ├── hints.md
                |   └── source.md (required if there are third-party sources)